srtcore/queue.cpp

```diff
@@ -492,6 +494,10 @@ CSndQueue::~CSndQueue()
     delete m_pSndUList;
 }
 
+#if ENABLE_LOGGING
+    int CSndQueue::m_counter = 0;
+#endif
```

Comment:
IF_HEAVY_LOGGING

Reply:
Unfortunately, it can't be used here. It's `ENABLE_LOGGING`, not `ENABLE_HEAVY_LOGGING`.